### PR TITLE
Feature/add kaios manifest

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

File: gem/views.py [hunk 1]
```diff
@@ -280,6 +281,32 @@ def get(self, request, comment_pk):
...
 
+class KaiOSManifestView(View):
+    def get(self, request):
+        manifest = {
+            "version": "1.0.0",
+            "name": "Springster App",
+            "description": "An app providing information to girls",
+            "launch_path": "/",
+            "icons": {
+                "56": "/gem/statc/img/icons/next.png",
```
Comment (miltontony) [inline gem/views.py]: This well probably be something like `/static/img/icons/next.png`

File: gem/views.py [hunk 2]
```diff
@@ -280,6 +281,32 @@ def get(self, request, comment_pk):
         })
 
 
+class KaiOSManifestView(View):
+    def get(self, request):
```
Comment (miltontony) [inline gem/views.py]: Almost.. you'll need to also change the headers as stipulated in the docs, so it might not necessarily be this method to use.

Comment (codiebeulaine) [reply gem/views.py]: hmmm, ok

File: gem/views.py (hunk 1 above)
Comment (codiebeulaine) [reply gem/views.py]: ok

File: gem/views.py [hunk 3]
```diff
@@ -280,6 +286,35 @@ def get(self, request, comment_pk):
...
+            },
+            "locales": {
+                "en": {
+                    "name": "Springster",
+                    "description": "An app providing information to girls"
+                }
+            },
+            "default_locale": "en"
+        }
+        response = HttpResponse(json.dumps(manifest))
```
Comment (miltontony) [inline gem/views.py]:
```suggestion
        response = HttpResponse(json.dumps(manifest), content_type='application/x-web-app-manifest+json')
```

File: gem/views.py [hunk 4]
```diff
@@ -280,6 +286,35 @@ def get(self, request, comment_pk):
...
+        response.headers['Content-Type'] = 'application/x-web-app-manifest+json'
```
Comment (miltontony) [inline gem/views.py]:
```suggestion
```

**Pull request merged**